data/names/ru.json

```diff
@@ -8679,7 +9296,1005 @@
       "Янира",
       "Янита",
       "Ясмин",
-      "Яхайра"
+      "Яхайра",
+      [ "Аарон", "Aaron" ],
+      [ "Августин", "Augustine" ],
+      [ "Агнес", "Agnes" ],
+      [ "Адалин", "Adaline" ],
+      [ "Адам", "Adam" ],
+      [ "Аделл", "Adell" ],
+      [ "Аджа", "Aja" ],
+      [ "Адриан", "Adrian" ],
+      [ "Азалия", "Azalee" ],
+      [ "Ай", "Ai" ],
+      [ "Айдэ", "Aide" ],
+      [ "Айко", "Aiko" ],
+      [ "Айриш", "Irish" ],
+      [ "Акико", "Akiko" ],
+      [ "Алан", "Alane" ],
+      [ "Алейда", "Aleida" ],
+      [ "Александр", "Alexander" ],
+      [ "Ален", "Alaine" ],
+      [ "Алеша", "Aleshia" ],
+      [ "Алида", "Alida" ],
+      [ "Алика", "Alica" ],
+      [ "Даймонд", "Diamond" ],
+      [ "Альберт", "Albert" ],
+      [ "Альма", "Alma" ],
+      [ "Ампаро", "Amparo" ],
+      [ "Ан", "An" ],
+      [ "Ангелена", "Angelena" ],
+      [ "Анджелес", "Angeles" ],
+      [ "Анджелин", "Angelyn" ],
+      [ "Анджело", "Angelo" ],
+      [ "Эндрю", "Andrew" ],
+      [ "Анжеле", "Angele" ],
+      [ "Анжелита", "Angelita" ],
+      [ "Анжелия", "Angelia" ],
+      [ "Аннали", "Annalee" ],
+      [ "Анналиса", "Annalisa" ],
+      [ "Аннамае", "Annamae" ],
+      [ "Аннемари", "Annemarie" ],
+      [ "Аннетта", "Annetta" ],
+      [ "Аннмари", "Annmarie" ],
+      [ "Антионетта", "Antionette" ],
+      [ "Антониетта", "Antonietta" ],
+      [ "Антонио", "Antonio" ],
+      [ "Аполония", "Apolonia" ],
+      [ "Арделл", "Ardell" ],
+      [ "Ализ", "Alease" ],
+      [ "Арлеан", "Arlean" ],
+      [ "Артур", "Arthur" ],
+      [ "Асусена", "Azucena" ],
+      [ "Аундреа", "Aundrea" ],
+      [ "Ауреа", "Aurea" ],
+      [ "Афтон", "Afton" ],
+      [ "Аяко", "Ayako" ],
+      [ "Базилия", "Basilia" ],
+      [ "Бао", "Bao" ],
+      [ "Бев", "Bev" ],
+      [ "Белл", "Bell" ],
+      [ "Белль", "Belle" ],
+      [ "Бёрилл", "Beryl" ],
+      [ "Бёрнадин", "Bernadine" ],
+      [ "Бёрнардин", "Bernardine" ],
+      [ "Бетанн", "Bethann" ],
+      [ "Бетси", "Betsey" ],
+      [ "Беттианн", "Bettyann" ],
+      [ "Беула", "Beula" ],
+      [ "Блейк", "Blake" ],
+      [ "Бобетт", "Bobette" ],
+      [ "Бок", "Bok" ],
+      [ "Бонг", "Bong" ],
+      [ "Боула", "Beaulah" ],
+      [ "Брайан", "Brian" ],
+      [ "Брайанна", "Bryanna" ],
+      [ "Бреана", "Breana" ],
+      [ "Бригида", "Brigida" ],
+      [ "Бриджитт", "Bridgett" ],
+      [ "Брэндон", "Brandon" ],
+      [ "Бэрри", "Berry" ],
+      [ "Вай", "Wai" ],
+      [ "Вайолет", "Violet" ],
+      [ "Вал", "Val" ],
+      [ "Валари", "Valarie" ],
+      [ "Вален", "Valene" ],
+      [ "Валентайн", "Valentine" ],
+      [ "Валерий", "Valeri" ],
+      [ "Ван", "Van" ],
+      [ "Ваня", "Vania" ],
```

Comment:
I just wanted to note how odd the transcription is. It is *correct*, and the fact that it's correct feels really odd.

For context: this seems to be a Portuguese name, spelled Vânia, and [every](https://ru.wikipedia.org/wiki/%D0%A4%D0%B5%D1%80%D0%BD%D0%B0%D0%BD%D0%B4%D0%B5%D1%88,_%D0%92%D0%B0%D0%BD%D1%8F) [single](https://ru.wikipedia.org/wiki/%D0%9A%D0%B8%D0%BD%D0%B3,_%D0%92%D0%B0%D0%BD%D1%8F) [entry](https://ru.wikipedia.org/wiki/%D0%9C%D0%B5%D0%BB%D0%BB%D0%BE,_%D0%90%D0%BD%D0%BD%D0%B0_%D0%92%D0%B0%D0%BD%D1%8F) I've seen with it in Russian (including the last one, of an Italian volleyball player) spells it out like a colloquial, casual dimunitive of the Russian name Ivan.